.docker/000-default.conf

```diff
@@ -10,6 +10,19 @@
     PassEnv TRUSTED_PROXIES
     PassEnv TRUSTED_HOSTS
```

Comment:
I wonder where `TRUSTED_HOSTS` is actually used. I couldn't find a usage in the whole repo.